versions.html

```diff
@@ -20,20 +20,20 @@ <h1> PyTorch Documentation </h1>
         <li class="toctree-l1">
           <a class="reference internal" href="main/">main (unstable)</a>
         </li>
+        <li class="toctree-l1">
+          <a class="reference internal" href="0.13.0/">v0.9.0 (release candidate)</a>
+        </li>
         <li class="toctree-l1">
           <a class="reference internal" href="0.12.0/">v0.12.0 (stable release)</a>
         </li>
-       <li class="toctree-l1">
+        <li class="toctree-l1">
```

Comment:
not sure if it matter, but can we remove extra space in the front?

Reply:
So this is actually removing the additional space that was there before to make this line consistent with how we indent the other `<li` lines